### PR TITLE
Daniel joo UI

### PR DESCRIPTION
**Pull request by danielsjoo**

### CODE REVIEW

**Comment by danielsjoo** [comment]
screenshots in case you don't want to run it

Capture 3
Capture 2
Capture1

File: src/App.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ class App extends React.Component<{}, State, {}> {
   constructor(props: {}) {
     super(props);
     this.state = {
-      isLoggedIn: false,
+      isLoggedIn: true,
```
Comment (loafyyy) [inline src/App.tsx:25]: Why did you change this?

Comment (loafyyy) [reply src/App.tsx:25]: I think you're supposed to change the state programmatically by logging in?

File: src/components/LandingPages/LandingPage.tsx [hunk 2]
```diff
@@ -1,15 +1,58 @@
 import React, { Component } from 'react';
 import { Helmet } from 'react-helmet';
+import { Container, Row, Col } from 'react-bootstrap';
+import aangImage from '../../static/images/aang.jpg';
+
+function AangCard(props) {
```
Comment (loafyyy) [inline src/components/LandingPages/LandingPage.tsx:6]: Good use of components/helper functions (so you didn't need to copy and paste this code like 6 times)!

**Review by loafyyy — APPROVED** [review]
looks good to me (LGTM) -- just have one comment though about why you changed the state in App.tsx

**Comment by danielsjoo** [comment]
@loafyyy I made that change in the app file so that I could get past the login page into the landing page without implementing the login code.

**Comment by loafyyy** [comment]
> @loafyyy I made that change in the app file so that I could get past the login page into the landing page without implementing the login code.

Ok, that's fine for the sake of this training module (but don't do that for the real code).
